Shastak/BulkThumbnailCreator/Modules/ImageHandler.cs

```diff
@@ -0,0 +1,229 @@
+
+                    imageNumber++;
+                    processedFiles++;
+                    filesDone++;
+
+                    imageToRename.Dispose();
+
+                    BTCLogger.WriteLocalLogs($"File {Files[i].Name} is renamed succesfully");
+                }
+
+                // if some exception is catched (e.g. file is not an image), then the file is skipped and amount of such files is incremented
+                catch (Exception ex)
+                {
+                    BTCLogger.WriteLocalLogs($"Error during resizing {Files[i].Name}. Error message: {ex.Message}");
+                    skippedFiles++;
+                    filesDone++;
+                }
+
+                // showing the progress of processing
+                finally
+                {
+                    DisplayProgress.Show(totalFiles, filesDone, ref length);
+                    Console.CursorLeft = DisplayProgress.CursorOffset;
+                }
+            }
+
+            // displaying operation results
+            DisplayOperationResult(isOperationAborted, "Operation is completed successfully!", processedFiles, skippedFiles);
+        }
+
+        public static void GetName(out string name)
+        {
+            // asking user for new name for the images
+            TextColorizer.WriteTextInColor("\nPlease enter new name for the files: ", ConsoleColor.Yellow, false);
+            name = Console.ReadLine().Trim();
+
+            Console.WriteLine();
+        }
+
+        public static void GetSize(out ushort width, out ushort height)
+        {
+            // asking user for size in pixels for each dimension
+            TextColorizer.WriteTextInColor("\nPLEASE PAY ATTENTION!", ConsoleColor.Red);
+            Console.WriteLine("To avoid huge usage of your hard drive memory the maximum size for one dimesion equals to 4096 pixels.");
+
+            TextColorizer.WriteTextInColor("\nPlease enter preferable width: ", ConsoleColor.Yellow, false);
+            width = ParsingChecker.SetPixels();
+
+            TextColorizer.WriteTextInColor("Please enter preferable heigth: ", ConsoleColor.Yellow, false);
+            height = ParsingChecker.SetPixels();
+
+            Console.WriteLine();
+        }
+
+        public static void GetFiles()
+        {
+            // getting files from the source directory and counting amount of files
+            DirectoryInfo folder = new DirectoryInfo(DirectoryHandler.SourceFiles);
+            Files = folder.GetFiles();
+            int filesCount = Files.Length;
+
+            // if there are no files in the source folder show a warning message for user to add files
+            while (filesCount == 0)
+            {
+                TextColorizer.WriteTextInColor(
+                    "Your source folder doesn't contain any files! \n" +
+                    "Please add some files in the source folder and press any key to continue. \n",
```

Comment:
Better to move message texts to separate class e.g. Constants

Reply:
fixed by adding Constants.cs class